compute_tools/src/compute.rs

```diff
@@ -347,6 +364,8 @@ impl ComputeNode {
                     .map_err(|_| anyhow::anyhow!("invalid connstr"))?;
 
                 let mut client = Client::connect(zenith_admin_connstr.as_str(), NoTls)?;
+                // Disable forwarding so that users don't get a cloud_admin role
+                client.simple_query("SET neon.forward_ddl = false")?;
```

Comment:
Will these roles be visible in the console UI?

Reply:
Nope, they are system roles